rotkehlchen/db/upgrades/v36_v37.py

```diff
@@ -213,13 +214,52 @@ def _fix_kraken_eth2_events(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _fix_kraken_eth2_events')
 
 
+def remove_ftx_data(write_cursor: 'DBCursor') -> None:
+    """Removes FTX-related settings from the DB"""
+    write_cursor.execute(
+        'DELETE FROM user_credentials WHERE location IN (?, ?)',
+        (Location.FTX.serialize_for_db(), Location.FTXUS.serialize_for_db()),
+    )
+    write_cursor.execute(
+        'DELETE FROM user_credentials_mappings WHERE credential_location IN (?, ?)',
+        (Location.FTX.serialize_for_db(), Location.FTXUS.serialize_for_db()),
+    )
+    write_cursor.execute(
+        'DELETE FROM used_query_ranges WHERE name LIKE ? ESCAPE ?;',
+        (f'{str(Location.FTX)}\\_%', '\\'),
+    )
+    write_cursor.execute(
+        'DELETE FROM used_query_ranges WHERE name LIKE ? ESCAPE ?;',
+        (f'{str(Location.FTXUS)}\\_%', '\\'),
+    )
+    non_syncing_exchanges_in_db = write_cursor.execute(
+        'SELECT value FROM settings WHERE name="non_syncing_exchanges"',
+    ).fetchone()
+    if non_syncing_exchanges_in_db is not None:
+        ftx_locations = (Location.FTX.serialize(), Location.FTXUS.serialize())
```

Comment:
why create a variable with this?

```suggestion
```

Reply:
I remember that if some constant is reused in a loop, we try to define the constant out of the loop scope. That's why I defined it separately. But surely it is not something mandatory.

Reply:
Some variable. Not a constant. If it's a constant the compiler/interpreter takes care of that.